notebooks/tutorial/2 - Download data.ipynb

```diff
@@ -89,23 +69,36 @@
    "outputs": [],
    "source": [
     "import os\n",
-    "from pureskillgg_dsdk.exchange import download_dataexchange_dataset_revision"
+    "import requests\n",
+    "import zipfile"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c63629e9",
-   "metadata": {
-    "scrolled": true
-   },
+   "id": "dc4195b1",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# Choose our Data Science collection path as the location to save\n",
-    "output_path = os.environ.get('PURESKILLGG_TOME_DS_COLLECTION_PATH')\n",
+    "# # Choose our Data Science collection path as the location to save\n",
+    "dataset_sample_urls = [\n",
+    "    \"https://d1ewbp317vsrbd.cloudfront.net/91b4d9a6-26a7-4c2e-838f-ba8d86c2b8b2.zip\",\n",
+    "    \"https://d1ewbp317vsrbd.cloudfront.net/4f6a3d09-99e8-476a-ba30-c7ee1992a276.zip\"\n",
+    "]\n",
+    "\n",
+    "ds_collection_path = os.environ.get('PURESKILLGG_TOME_DS_COLLECTION_PATH')\n",
+    "\n",
+    "for index, url in enumerate(dataset_sample_urls):\n",
+    "    output_filename = os.path.join(ds_collection_path,f\"sample_{index}.zip\")\n",
+    "    if os.path.exists(output_filename):\n",
+    "        continue\n",
     "\n",
-    "# Download the data\n",
-    "download_dataexchange_dataset_revision(output_path, dataset_id)"
+    "    response = requests.get(url)\n",
+    "    with open(output_filename, \"wb\") as f:\n",
+    "        f.write(response.content)\n",
```

Comment:
We can just skip the part where we save the zip file to disk: https://stackoverflow.com/a/2463819

Reply:
This is such a small amount of data I'm not sure it matters

Reply:
Also note that the way you've written this, if there was an error extracting the zip file and they reran the loop it would still assume the zip file was extracted correctly. So I recommend just not making this one "smart"

Reply:
The old one took 20 minutes to download, this one takes less than 10 seconds so it is trivial to overwrite, i guess :)

Reply:
Fix'd